src/pre_process/m_initial_condition.fpp

```diff
@@ -190,9 +196,13 @@ contains
                 elseif (patch_icpp(i)%geometry == 5) then
                     call s_ellipse(i, patch_id_fp, q_prim_vf)
 
-                    ! Isentropic vortex patch
+                    ! Unimplemented patch
                 elseif (patch_icpp(i)%geometry == 6) then
-                    call s_isentropic_vortex(i, patch_id_fp, q_prim_vf)
+                    call s_int_to_str(i, iStr)
+
+                    call s_mpi_abort('Unimplemented choice of geometry 6'// &
```

Comment:
you have two calls to `mpi_abort` for the same thing, one in `m_initial_condition` and one in `m_check_patches`.... only one of these needs to exist (whichever the code calls first). the second can be turned into a code comment.

Reply:
Removed the one in `m_initial_condition` since it occurs second.